src/test/javascript/spec/component/learning-goals/create-learning-goal.component.spec.ts

```diff
@@ -80,7 +80,8 @@ describe('CreateLearningGoal', () => {
         createLearningGoalComponentFixture.detectChanges();
```

Comment:
You missed a codacy issue here where you need to remove `fakeAsnyc`

Reply:
Thanks, fixed.